camera_to_hls/README.md

```diff
@@ -0,0 +1,52 @@
+# Membrane Demo - Camera Video to HLS
+
+This project demonstrates capturing camera video and converting it to HLS stream, which is then can be used wherever you want :)
+
+## Prerequisites
+
+You must have following packages installed on your system:
+
+- ffmpeg 4.\*
```

Comment:
I think openssl is also required. Native dependencies probably won't compile without it